packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -868,23 +842,6 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     }
 
     public get storage(): IDocumentStorageService {
-        // This code is plain wrong. It lies that it never returns undefined!!!
```

Comment:
I'd keep this comment - it's unrelated to packing

Reply:
Since Jatin's check in from 10 months ago, the storage can not be undefined. 

[https://github.com/microsoft/FluidFramework/commit/794fa4b2ad72a96d9cf31a79bc35b25d4cc60ca6](https://github.com/microsoft/FluidFramework/commit/794fa4b2ad72a96d9cf31a79bc35b25d4cc60ca6)